templates/lookup/ifupdown__dynamic_packages.j2

```diff
@@ -16,6 +16,4 @@
 {%     endif %}
 {%   endfor %}
 {% endif %}
-{% if ifupdown__tpl_packages %}
 {{ ifupdown__tpl_packages | unique | to_yaml }}
-{% endif %}
```

Comment:
Yes, I did. Worked without issues.

> changed: [keyhole.pubsrv.ypid.de] => (item=[u'resolvconf', u'rdnssd', u'bsdutils', u'rsync', u'bridge-utils'])

The packages where correctly picked up as it seems to me.